notifications/rest/bindings/create-binding/create-binding.rb

```diff
@@ -0,0 +1,14 @@
+# Get twilio-ruby from twilio.com/docs/ruby/install
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = '{{ auth_token }}'
+client = Twilio::REST::Client.new account_sid, auth_token
+
+# Retrieve a valid notification service
+service = client.notifications.v1.services('ISxxx')
+
+binding = service.bindings.create(:endpoint => "xxx", :identity => 'bob', :binding_type => 'gcm', :address => 'xxx', :tag => 'new user') 
```

Comment:
Based on [this style guide](https://github.com/bbatsov/ruby-style-guide), line is too long. You could align parameters to the first parameter:

```
binding = service.bindings.create(endpoint: 'xxx',
                                  identity: 'bob',
                                  binding_type: 'gcm',
                                  address: 'xxx',
                                  tag: 'new user')
```
or
```
binding = service.bindings.create(
  endpoint: 'xxx',
  identity: 'bob',
  binding_type: 'gcm',
  address: 'xxx',
  tag: 'new user'
)
```